Doc/library/socketserver.rst

```diff
@@ -176,7 +176,7 @@ partially finished requests and to use :mod:`selectors` to decide which
 request to work on next (or whether to handle a new incoming request).  This is
 particularly important for stream services where each client can potentially be
 connected for a long time (if threads or subprocesses cannot be used).  See
-:mod:`asyncio` for another way to manage this.
+:mod:`asyncore` for another way to manage this.
```

Comment:
Can't you still recommend `asyncio` even with the revert?

Reply:
Yes, it's possible, but I wanted to use a simple "git revert" command. Moreover, Miro wants to get it in 3.11.0a3, and Pablo suggests to merge the PR today, so I simply merged my PR.